Controls/Mail/Mailman.c

```diff
@@ -13,6 +16,26 @@
  */
 void depositLetter(void) {
 	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+
+	// Generate Random Number
+	char randomNumberString[4];
+	sprintf(randomNumberString, "%d", rand() % 9999 + 1);
+
+	// Concatenate message to mailbox
+	strcpy(mailbox, "Get your own free pizza at ");
+	strcat(mailbox,  randomNumberString);
+	strcat(mailbox, " Gullible St!");
```

Comment:
Instead of using this sequence of copies and catenates, can you think of a way to do all of this string manipulation just using sprintf? (This is a nice way to do it, though).

Reply:
Pushed commit to combine the four lines of code that added the message to the mailbox.